packages/web/src/form/form.js

```diff
@@ -66,15 +66,22 @@ const inputTagProps = (props) => {
   // any errors on this field
   const validationError = errors[props.name]
 
-  // get errorStyle/errorClassName and replace style/className if present
-  const { errorClassName, errorStyle, ...tagProps } = props
+  // get error/validStyle and error/validClassName and replace style/className if present
+  const { errorClassName, errorStyle, validClassName, validStyle, ...tagProps } = props
```

Comment:
This should fix these linting errors:
```suggestion
  const {
    errorClassName,
    errorStyle,
    validClassName,
    validStyle,
    ...tagProps
  } = props
```